core/templates/components/code-mirror/codemirror-mergeview.component.ts

```diff
@@ -27,14 +27,22 @@ import { downgradeComponent } from '@angular/upgrade/static';
 export class CodemirrorMergeviewComponent implements
   AfterViewInit, OnInit, OnChanges {
   @Input() options = {};
-  @Input() leftValue;
-  @Input() rightValue;
-  codeMirrorInstance: CodeMirror.MergeView.MergeViewEditor;
+  // Below properties have been assigned as empty string if undefined.
+  // See function 'ngAfterViewInit' for reference.
+  @Input() leftValue: string | undefined;
+  @Input() rightValue: string | undefined;
+  // Non-null assertion operator(!) to ignore
+  // the case when CodeMirror is undefined or null.
```

Comment:
I don't think we need to explain typescript features in comments. The wiki is present for that. We can send a mail to oppia-dev with the wiki link (And maybe a TL;DR version too). Just put in when this can be undefined. (This cannot be null).

Reply:
Updated, please take a look!